src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { MenuItem, Select } from "@material-ui/core";
+import React from "react";
+import i18n from "../utils/i18n";
+import { DropdownForm, DropdownItem } from "./GenericDropdown";
+
+export interface DropdownProps<Value extends string = string> {
+    className?: string;
+    items: DropdownItem[];
+    onChange: (value: Value | undefined) => void;
+    label?: string;
+    value?: Value;
+    hideEmpty?: boolean;
+}
+
+interface SelectProps {
+    className?: string;
+    label?: string;
+}
+
+const SelectWrapper: React.FC<SelectProps> = React.memo(props => {
+    const { className, label, children } = props;
+    return label ? (
+        <DropdownForm className={className} label={label}>
+            {children}
+        </DropdownForm>
+    ) : (
+        <div className={className}>{children}</div>
+    );
+});
+
+export const Dropdown: React.FC<DropdownProps> = React.memo(props => {
+    const { items, value, onChange, label, hideEmpty, className } = props;
+
+    const selectValue =
+        value === undefined || !items.map(item => item.value).includes(value) ? "" : value;
+
+    return (
+        <SelectWrapper className={className} label={label}>
+            <Select
+                data-cy={label}
+                value={selectValue}
+                onChange={ev => onChange((ev.target.value as string) || undefined)}
+                MenuProps={{
+                    getContentAnchorEl: null,
+                    anchorOrigin: { vertical: "bottom", horizontal: "left" },
+                }}
```

Comment:
Can we make onChange and MenuProps immutable?

Reply:
As this is really minor, leaving it for now.

We might revisit them as soon as the component is used on other projects.

Maybe we could migrate the multiple dropdown on SP and revisit this minor things and maybe other features. See https://github.com/EyeSeeTea/project-monitoring-app/blob/development/src/components/dropdown/MultipleDropdown.tsx